Make background blur keymap agnostic

ndedakaduki opened the pull request:
The previous implementation of the background blur depended on the user having the default VSCode keymaps in order to trigger the background blur. This implementation fixes that by using observers to detect when the command center has been triggered and set the background accordingly.

glennraya commented:
Please give me some time to check this, got some other things to do. 🙏🏼🙂

ndedakaduki commented:
No problem, take your time.

glennraya commented:
<img width="1955" alt="Screenshot 2024-08-09 at 11 06 21 AM" src="https://github.com/user-attachments/assets/c5e94213-ad74-4cd0-b25d-abbdc7b913fd">

It appears the background blur, doesn't work.

ndedakaduki commented:
I updated the name of the `div` in my implementation of the customizations, I've reverted it to the original name.
The background blur should now be working.

glennraya commented:
Still doesn't work. I remapped the "quick look" (cmd+p) to something else, and the background blur doesn't show, even after this update. I enabled the extension, reloaded it, and still no luck.

glennraya commented:
Actually, even if I remapped the cmd+p key to something else, the background blur still works.

ndedakaduki commented:
That's odd, it should be working. Let me take another, more thorough, look at it.

ndedakaduki commented:
![Screenshot 2024-08-09 142018](https://github.com/user-attachments/assets/56682279-66c8-4910-8ca9-64643b23c809)
I spun up a sandbox and it's behaving as expected.
Might you be able to hop on a quick call, I'm unable to reproduce this on my end.

glennraya commented:
Probably this has something to do with the local environment we have, you're on Windows, and I'm on Mac. So results might differ on different local dev environment.
